### ISSUE TITLE
Let annotatations span on several lines

### ISSUE THREAD

**Issue by fedinskiy** [quarkus-qe/quarkus-test-framework#644]
Right now, if we have something like that in the code:
```
    @GitRepositoryQuarkusApplication(repo = "https://github.com/quarkusio/todo-demo-app.git",
            artifact = "todo-backend-1.0-SNAPSHOT-runner.jar",
            mavenArgs = ARGS)
```
(max line length is 89, easy to read and understand)
then our formatter will glue it in one line:
`@GitRepositoryQuarkusApplication(repo = "https://github.com/quarkusio/todo-demo-app.git", artifact = "todo-backend-1.0-SNAPSHOT-runner.jar", mavenArgs = ARGS)`
(max line length is 158 characters, harder to read and understand)

We should change the formatting rules for both FW and TS

**Comment by MOue25** [comment]
Closing the linked PR (quarkusio/quarkus#55509)  the fix needs a full codebase reformat that isn't backport friendly, so it won't be resolving this. @fedinskiy We need to discuss whether we should create our own formatter config file in our projects.  Couldn't remove the PR link from my side

**Comment by fedinskiy** [comment]
@MOue25 I would say, that style rules should be set separately by each team, and we should not directly depend on whatever upstream folks have.

Isn't there a way to overload this exact parameter in our pom(s)?

**Comment by MOue25** [comment]
@fedinskiy Agreed  tying our style to upstream isn't ideal. To give the background while investigating, the root cause was the alignment_for_arguments_in_annotation=0 setting in the Quarkus upstream repo. It looked like a one line fix at first (quarkusio/quarkus#55509), but because the formatter is CI validated, the change also required reformatting the entire upstream codebase  which was declined due to the backport conflicts it would create. So the upstream route is a dead end.
I'll look into overriding the formatter config in our own pom(s). should i create an issue also in TS since we want this style in all our projects?

**Comment by fedinskiy** [comment]
@MOue25 let's start with a PR for the framework only, to get some feedback about the chosen direction
